contracts/Bridge/BaseXVSProxyOFT.sol

```diff
@@ -229,27 +240,73 @@ abstract contract BaseXVSProxyOFT is Pausable, ExponentialNoError, BaseOFTV2 {
         _unpause();
     }
 
+    /**
+     * @notice A public function to sweep accidental ERC-20 transfers to this contract. Tokens are sent to user
+     * @param token_ The address of the ERC-20 token to sweep
+     * @param to_ The address of the recipient
+     * @param amount_ The amount of tokens needs to transfer
+     * @custom:event Emits SweepToken event
+     * @custom:error Throw InsufficientBalance if amount_ is greater than the available balance of the token in the contract
+     * @custom:access Only Owner
+     */
+    function sweepToken(IERC20 token_, address to_, uint256 amount_) external onlyOwner {
```

Comment:
I think we should add this function to the list `xvsBridgeMethods` in the file `deploymentConfig.ts`

Reply:
[4ef1a4c](https://github.com/VenusProtocol/token-bridge/pull/8/commits/4ef1a4c27fd824d8d4915f44a0c6c37d714a07c0)